lua/astronvim/mappings.lua

```diff
@@ -376,5 +376,6 @@ maps.n["<leader>ut"] = { ui.toggle_tabline, desc = "Toggle tabline" }
 maps.n["<leader>uu"] = { ui.toggle_url_match, desc = "Toggle URL highlight" }
 maps.n["<leader>uw"] = { ui.toggle_wrap, desc = "Toggle wrap" }
 maps.n["<leader>uy"] = { ui.toggle_syntax, desc = "Toggle syntax highlight" }
+if vim.fn.has "nvim-0.9" == 1 then maps.n["<leader>uh"] = { ui.toggle_foldcolumn, desc = "Toggle foldcolumn" } end
```

Comment:
```suggestion
maps.n["<leader>uh"] = { ui.toggle_foldcolumn, desc = "Toggle foldcolumn" }
```

This doesn't need to rely on Neovim 0.9. `foldcolumn` is still a feature that could be toggled if the user wants, it just won't be as pretty